scripts/setupvars/setupvars.sh

```diff
@@ -45,24 +45,32 @@ if [ -e "$INSTALLDIR/runtime" ]; then
     if [ -e "$HDDL_UNITE_DIR" ]; then
         export LD_LIBRARY_PATH=$HDDL_UNITE_DIR/lib:$HDDL_UNITE_DIR/thirdparty/XLink/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
     fi
-fi
 
-if [ -e "$INSTALLDIR/runtime/3rdparty/tbb" ]; then
-    if [[ "$OSTYPE" == "darwin"* ]]; then
-        export DYLD_LIBRARY_PATH=$INSTALLDIR/runtime/3rdparty/tbb/lib:${DYLD_LIBRARY_PATH:+:$DYLD_LIBRARY_PATH}
-    fi
-    export LD_LIBRARY_PATH=$INSTALLDIR/runtime/3rdparty/tbb/lib:${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH}
-
-    if [ -e "$INSTALLDIR/runtime/3rdparty/tbb/lib/cmake/TBB" ]; then
-        export TBB_DIR=$INSTALLDIR/runtime/3rdparty/tbb/lib/cmake/TBB
-    elif [ -e "$INSTALLDIR/runtime/3rdparty/tbb/lib/cmake/tbb" ]; then
-        export TBB_DIR=$INSTALLDIR/runtime/3rdparty/tbb/lib/cmake/tbb
-    elif [ -e "$INSTALLDIR/runtime/3rdparty/tbb/lib64/cmake/TBB" ]; then
-        export TBB_DIR=$INSTALLDIR/runtime/3rdparty/tbb/lib64/cmake/TBB
-    elif [ -e "$INSTALLDIR/runtime/3rdparty/tbb/cmake" ]; then
-        export TBB_DIR=$INSTALLDIR/runtime/3rdparty/tbb/cmake
-    else
-        echo "TBB_DIR directory is not defined automatically by setupvars.sh. Please, set it manually to point to TBBConfig.cmake"
+    if [ -e "$INSTALLDIR/runtime/3rdparty/tbb" ]; then
+        tbb_lib_path=$INSTALLDIR/runtime/3rdparty/tbb/lib
+        if [ -d $tbb_lib_path/$system_type ]; then
+            lib_path=$(find $tbb_lib_path/$system_type -name "libtbb*" | head -n1)
+            if [ -n $lib_path ]; then
+                tbb_lib_path=$(dirname $lib_path)
```

Comment:
I see the following in the intel64, ia32:
```
$ ls -1 tbb2017_20161004oss/lib/intel64
gcc4.1
gcc4.4
gcc4.7
```
I suppose we need to select higher version of gcc?